projects/ontimize-web-ngx/src/lib/components/table/o-table.component.ts

```diff
@@ -2562,6 +2562,17 @@ export class OTableComponent extends AbstractOServiceComponent<OTableComponentSt
     return Util.isDefined(column.definition) && Util.isDefined(column.definition.contentAlign) ? 'o-' + column.definition.contentAlign : '';
   }
 
+
+  public getGroupHeadeCellAlignClass(column: string): string[] {
```

Comment:
```suggestion
  public getGroupHeaderCellAlignClass(column: string): string[] {
```
Fix typo in method name